test/testdata/sample3.txt

```diff
@@ -0,0 +1,64 @@
+01,103100195,103100195,220919,2112,4,,,2/   02,103100195,103100195,1,220919,2400,USD,2/
+03,1111111,,010,-3500,,,015,-3600,,,040,-3500,,,060,-3600,,,100,3100,3,,400,3200,3,,/
+16,142,2500,Z,,,TRANSFER   PAYPAL             PPD/ 16,142,500,Z,,,TRANSFER   MSPBNA BANK        PPD/
+16,142,100,Z,,,Ext Trnsfr JPMorgan Chase     PPD/
+16,451,2500,Z,,,111111     ACH_SETL           1111111111                    111111111111111        /
+88,       1111111111/
+16,451,600,Z,,,111111     ACH_SETL           1111111111                    111111111111111         /
+88,      1111111111/
+16,451,100,Z,,,Ext Trnsfr JPMorgan Chase     1111111111                    111111111111111         /
+88,      11111111111/         49,-1600,11/  03,1111111,,010,00,,,015,00,,,040,00,,,060,00,,,100,00,0,,400,00,0,,/
+49,00,2/  03,1111111,,010,00,,,015,00,,,040,00,,,060,00,,,100,00,0,,400,00,0,,/ 49,00,2/
```

Comment:
I'm not sure what happened when this sample file got copied over, but these three records shouldn't all be on the same line. Same goes for the line above it and several others in this file. As far as I can tell there should only ever be a single record per line in this format, so multiple records per line shouldn't even be a thing we need to test against.

Can we re-copy the sample file I sent over to correct this?

Reply:
Do you have it to git push somewhere? That would help save the original formatting.

Reply:
[bai2_cleaned.txt](https://github.com/moov-io/bai2/files/10393769/bai2_cleaned.txt)

Reply:
Parsing what we know will come through is a good first step.